rules/test-title-format.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const {visitIf} = require('enhance-visitors');
+const createAvaRule = require('../create-ava-rule');
+const util = require('../util');
+
+const create = context => {
+	const ava = createAvaRule();
+	let titleRegExp;
+	try {
+		if (context.options[0] && context.options[0].regexp) {
+			titleRegExp = new RegExp(context.options[0].regexp);
+		}
+	} catch (error) {
+		console.warn(error);
+	}
+
+	return ava.merge({
+		CallExpression: visitIf([
+			ava.isInTestFile,
+			ava.isTestNode,
+			ava.hasNoHookModifier
+		])(node => {
+			const requiredLength = ava.hasTestModifier('todo') ? 1 : 2;
+			const hasTitle = node.arguments.length >= requiredLength;
+
+			if (hasTitle && titleRegExp) {
+				const title = node.arguments[0];
+				if (title.type === 'Literal' && !titleRegExp.test(title.value)) {
+					context.report({
+						node,
+						message: 'Title doesn\'t match required format.'
```

Comment:
```suggestion
						message: 'The test title doesn\'t match the required format: '
```

And include the regex stringified.